### ISSUE TITLE
Wait: indicator missing text equivalent (1.1.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#868]
### Expected behavior

Indicator communicates via a text equivalent that the system is working or loading something. 
### Actual behavior

Someone who is using screenreader will hear an empty text element and not see the loading visual indicator. For a full page wait, this can be problematic when it take time to load and someone doesn't have any feedback that the system is loading something.
